tls/s2n_cipher_preferences.c

```diff
@@ -1073,6 +1075,10 @@ int s2n_cipher_preferences_init()
              * but the elliptic curves extension is always required. */
             if (cipher->minimum_required_tls_version >= S2N_TLS13) {
                 selection[i].ecc_extension_required = 1;
+                /* Valid TLS 1.3 Ciphers are 0x1301, 0x1302, 0x1303, 0x1304, 0x1305 */
+                bool is_tls13_cipher = cipher->iana_value[0] == 0x13 && cipher->iana_value[1] >= 0x01 && cipher->iana_value[1] <= 0x05;
+                S2N_ERROR_IF(!is_tls13_cipher, S2N_ERR_INVALID_CIPHER_PREFERENCES);
+                selection[i].tls13_required = true;
```

Comment:
looks like we use `1` for the truth value on bit fields, and you also compare this to `1` later.

suggest you use `1` for consistency over `true`